app/src/main/java/eu/kanade/tachiyomi/data/database/models/MangaImpl.kt

```diff
@@ -70,6 +70,8 @@ open class MangaImpl : Manga {
 
     override var last_chapter_number: Int? = null
 
+    override var similar_type_string: String = ""
+
```

Comment:
Do we really need to add something to the database for this?

Reply:
I was unsure how best to get it to the UI frontend if it is not appended here. It isn't committed to the database at all.